cmake/px4_add_common_flags.cmake

```diff
@@ -69,6 +69,7 @@ function(px4_add_common_flags)
 		-Wall
 		-Wextra
 		-Werror
+		-Wno-error=address
```

Comment:
Is this only for xtensa?

Reply:
I have compiled it without this compiler flag, and its successfull. I think i added this when I had nuttx issues. 
I will test on Tuesday on hardware.

Reply:
removed, compiled successfully, and running as expected.